deploy/api.dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.7
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
-RUN apt update && apt install nodejs -y
+RUN apt update && curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt install nodejs -y
```

Comment:
might be simpler to move this all to one line (i.e.,  ` ... && curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt install nodejs -y`)